recipes/oommf/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: oommf
+  version: "1.2b0_20160930b1"
+
+source:
+  git_url: https://github.com/fangohr/oommf
+
+build:
+  number: 0
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+    - curl     # [win]
+    - 7za      # [win]
+  run:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+
+test:    # [unix]
+  commands:   # [unix]
+    - oommf +version   # [unix]
+
+about:
+  home: http://math.nist.gov/oommf/
+  license_file: oommf/LICENSE
+  summary: 'Object Oriented MicroMagnetic Framework (OOMMF)'
+
+  description: |
+    The Object Oriented MicroMagnetic Framework (OOMMF) forms a
+    completely functional micromagnetics package. OOMMF is written in
+    C++. For portable user interfaces, it uses Tcl/Tk so that OOMMF
+    operates across a wide range of Unix, Windows, and MacOSX
+    platforms.
+
+  doc_url: http://math.nist.gov/oommf/doc/
+
+extra:
+  recipe-maintainers:
+    - fangohr
+    - mb1a15
+    - takluyver
+    - davidcortesortuno
+    - rpep
```

Comment:
I think we have addressed all queries now. Thank you @isuruf for the quick review of this, the thorough review, and the repeated feedback and help.